plenum/common/error_codes.py

```diff
@@ -1,3 +1,4 @@
 SOCKET_BIND_ERROR_ALREADY_IN_USE = 98
 WS_SOCKET_BIND_ERROR_ALREADY_IN_USE = 10048
-WS_SOCKET_BIND_ERROR_NOT_AVAILABLE = 10049
\ No newline at end of file
+WS_SOCKET_BIND_ERROR_NOT_AVAILABLE = 10049
+NO_SUCH_FILE_OR_DIRECTORY = 2
```

Comment:
Should we move these error codes to place where we are using them? Also, can we use python's module `errno` to  define error code numbers, so that those can be captured as same on windows as well.